src/Offer/ReadModel/JSONLD/PropertyPolyfillOfferRepository.php

```diff
@@ -264,6 +265,20 @@ function (array $json) {
         );
     }
 
+    private function fixMainImage(JsonDocument $jsonDocument): JsonDocument
```

Comment:
I would make the name of the method a bit more explicit. I think the names of the test methods describe very well what happens. For example `removeMainImageWhenMediaObjectIsEmpty`